install/charts/dir/apiserver/templates/deployment.yaml

```diff
@@ -36,6 +36,12 @@ spec:
           env:
             - name: DIRECTORY_LOGGER_LOG_LEVEL
               value: {{ .Values.log_level }}
+            {{- if .Values.publication }}
+            {{- if .Values.publication.scheduler_interval }}
+            - name: DIRECTORY_SERVER_PUBLICATION_SCHEDULER_INTERVAL
+              value: {{ .Values.publication.scheduler_interval }}
+            {{- end }}
+            {{- end }}
```

Comment:
are we not loading this from config already? not sure if we need this here.

Reply:
please extend the values.yaml helm chart file to include the publication config details as well. after that, the taskfile scheduler interval can simply override these default helm values

Reply:
we need this override for the tests, otherwise they will fail with the default 1h publication interval